CHANGELOG.md

```diff
@@ -92,6 +92,8 @@ This project uses the changelog in accordance with [keepchangelog](http://keepac
  - Added `script run hf_mf_hid_sim.lua` (@micsen)
  - Added `hf 15 sim --blocksize` - configure block size for simulation (@markus-oehme-pg40)
  - Added `hf 15 eload` - specify memory image for ISO15693 simulation (@markus-oehme-pg40)
+ - Added write command support to `hf 15 sim` (@markus-oehme-pg40)
+ - Added `hf 15 eview` and `hf 15 esave` - Retrieve emulator image for ISO15693 simulation (@markus-oehme-pg40)
```

Comment:
I forgot to mention last time, but we put all entries on the top of the list,  
so move your four entries to the top instead.
We also use,   "Changed",  "Fix",  "Added"  as the first word,   with <cmd> as second::
You write entry becomes:

"Changed `hf 15 sim` - now supports reader writes (@markus-oehme-pg40 )